modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -105,21 +105,27 @@ def make_request(endpoint:, action:, body:, key: nil) # rubocop:disable Metrics/
       end
       connection.options.timeout = @timeout
 
-      wsdl = log_duration(event: 'connection_wsdl_get', endpoint:) do
-        connection.get("#{Settings.bgs.url}/#{endpoint}?WSDL")
-      end
-      target_namespace = Hash.from_xml(wsdl.body).dig('definitions', 'targetNamespace')
-      response = log_duration(event: 'connection_post', endpoint:, action:) do
-        connection.post("#{Settings.bgs.url}/#{endpoint}", full_body(action:,
-                                                                     body:,
-                                                                     namespace: target_namespace),
-                        {
-                          'Content-Type' => 'text/xml;charset=UTF-8',
-                          'Host' => "#{@env}.vba.va.gov",
-                          'Soapaction' => "\"#{action}\""
-                        })
+      begin
+        wsdl = log_duration(event: 'connection_wsdl_get', endpoint:) do
+          connection.get("#{Settings.bgs.url}/#{endpoint}?WSDL")
+        end
+        target_namespace = Hash.from_xml(wsdl.body).dig('definitions', 'targetNamespace')
+        response = log_duration(event: 'connection_post', endpoint:, action:) do
+          connection.post("#{Settings.bgs.url}/#{endpoint}", full_body(action:,
+                                                                       body:,
+                                                                       namespace: target_namespace),
+                          {
+                            'Content-Type' => 'text/xml;charset=UTF-8',
+                            'Host' => "#{@env}.vba.va.gov",
+                            'Soapaction' => "\"#{action}\""
+                          })
+        end
+      rescue Faraday::TimeoutError, Faraday::ConnectionFailed => e
```

Comment:
@acovrig can you check this with your tooling? the EoF throws this connection failed.. so would like to check it that way.

Reply:
If I understand things correctly, then this does rescue from an EOF.